lib/rules/callback-return.js

```diff
@@ -10,6 +10,10 @@
 
 module.exports = {
     meta: {
+        deprecated: true,
+
+        replacedBy: [],
```

Comment:
Would it make sense to include the rule it's replaced by in the plugin?

Reply:
I went back and forth on that. Since we won’t be maintaining it, I’m a little worried about it getting out of sync (even though it’s maintained by one of us). Happy to defer to consensus on this, though.

Reply:
I think if we are recommending people use the plugin in our docs and release notes then it makes sense to be part of the console deprecation notice.

Reply:
:+1: Definitely not against that. Waiting on `eslint-plugin-node` to be released and will update this PR.

Reply:
This has been updated!